core/exchange_test.go

```diff
@@ -44,6 +44,9 @@ func TestCoreExchange_RequestHeaders(t *testing.T) {
 	expectedLastHeightInRange := to - 1
 	expectedLenHeaders := to - expectedFirstHeightInRange
 
+	_, err = cctx.WaitForHeightWithTimeout(30, 10*time.Second)
+	require.NoError(t, err)
```

Comment:
I think we need to wait for some blocks due to the `BroadcastModeBlock` being removed.